vscode/src/gh-copilot/instructions.ts

```diff
@@ -292,7 +318,24 @@ export function registerGhCopilotInstructionsCommand(
   context.subscriptions.push(
     vscode.commands.registerCommand(
       "qsharp-vscode.updateCopilotInstructions",
-      updateGhCopilotInstructionsCommand,
+      () => updateGhCopilotInstructionsCommand(true),
     ),
   );
+
+  // Also do a one-time prompt at startup
+  if (
+    context.globalState.get<boolean>(
```

Comment:
might want to make this `workspaceState` since the instructions files are workspace specific

Reply:
N/A since my last change